docs/install.md

```diff
@@ -81,6 +81,9 @@ In the build folder
 #On Linux
 > CXX=g++ cmake -DCMAKE_INSTALL_PREFIX=../install ../src/fringe
 
+# with anaconda (on Linux) 
+> CXX=g++ cmake -DCMAKE_INSTALL_PREFIX=../install ../src/fringe -DCMAKE_PREFIX_PATH=$CONDA_PREFIX
+
 #If "conda install gxx_linux-64 / clangxx_osx-64"
 > CXX=${CXX} cmake -DCMAKE_INSTALL_PREFIX=../install ../src/fringe
```

Comment:
CXX=${CXX} is redundant when using conda compilers since these are automatically set when the environment is activated. I would remove that.